mdoc/src/main/scala/mdoc/internal/markdown/Mod.scala

```diff
@@ -38,7 +48,16 @@ object Mod {
       ToString,
       Nest
     )
+
+  def parametric: List[String => Option[Mod]] =
```

Comment:
nit: I feel like the following solution is a bit nicer
```scala
import scala.util.Try
sealed abstract class Mod extends Product with Serializable
case object Passthrough extends Mod
val static = List[Mod](Passthrough)
case class Width(n: Int) extends Mod
case class Heigth(n: Int) extends Mod
val ToWidth = "width=(\\d+)".r
val ToHeight = "height=(\\d+)".r
object ToInt {
  def unapply(s: String): Option[Int] = Try(s.toInt).toOption
}

List("width=24", "height=42", "width=abc").map {
  case ToWidth(ToInt(n)) => Some(Width(n))
  case ToHeight(ToInt(n)) => Some(Heigth(n))
  case _ => None
} // List(Some(Width(24)), Some(Height(42)), None)
```

Reply:
Nice.

I've changed it very slightly, how does 21106b3 look?